Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/FacepunchMultiplayerMenu/FacepunchSteamworksController.cs

```diff
@@ -0,0 +1,42 @@
+﻿#if FACEPUNCH_STEAMWORKS
+using UnityEngine;
+using Steamworks;
+
+/// <summary>
+/// Singleton class to initialize Facepunch.Steamworks client and shotdown on application quit
+/// </summary>
+public class FacepunchSteamworksController : MonoBehaviour
+{
+	public static FacepunchSteamworksController facepunchSteamworksController;
+
+	private void Awake()
+	{
+		if (facepunchSteamworksController == null)
+		{
+			facepunchSteamworksController = this;
+			DontDestroyOnLoad(gameObject);
+
+			try
+			{
+				// Use your Steam appID here. 480 is the test SpaceWar app
+				SteamClient.Init(480);
```

Comment:
Maybe extract `480` into a const to be more descriptive?
```suggestion
				SteamClient.Init(STEAM_APP_ID);
```